app/models/worldwide_organisation.rb

```diff
@@ -84,7 +84,7 @@ def sponsoring_organisation
              format: "worldwide_organisation"
 
   def search_link
-    Whitehall.url_maker.worldwide_organisation_path(slug)
+    public_path
```

Comment:
Same here with the `search_link` method (I think it applies in other models too).